BraveWallet/WalletStrings.swift

```diff
@@ -1121,6 +1128,13 @@ extension Strings {
       value: "Gas Fee",
       comment: "A title displayed beside a number describing the cost of the transaction in ETH which is called Gas"
     )
+    public static let transactionFee = NSLocalizedString(
+      "wallet.transactionFee",
```

Comment:
I wonder if we could re-use `Strings.Wallet.transactionDetailsTxFeeTitle` here? Capitalization of `Fee` vs `fee` difference between the two views though.

Reply:
hmmmm to use string format and pass either copy of `Gas` or `Transaction` for different coin type transaction? in that case, i wll change the `Strings.Wallet.transactionDetailsTxFeeTitle` just to be `Strings.Wallet.Fee`